src/Microsoft.Health.Dicom.Api/Features/Filters/QueryModelStateValidatorAttribute.cs

```diff
@@ -19,7 +21,14 @@ public override void OnActionExecuting(ActionExecutingContext context)
         if (!context.ModelState.IsValid)
         {
             (string key, ModelStateEntry value) = context.ModelState.Where(x => x.Value.Errors.Count > 0).First();
-            throw new InvalidQueryStringValuesException(key, value.Errors[0].ErrorMessage);
+
+            string errorMessage = value.Errors[0].ErrorMessage;
+            if (Regex.IsMatch(errorMessage, @"<[^>]*>"))
```

Comment:
Can you make this Regex compiled. So we dont spend time.

You can make this a constant.
```
string pattern = @"<[^>]*>";
Regex regex = new Regex(pattern, RegexOptions.Compiled);
```

And then you can use.

```regex.IsMatch(errorMessage)```

Reply:
updated